maven-enforcer-plugin does not allow Java 1.8.0_05

Spring Boot fails to build after updating to Java version 1.8.0_05. The problem is that the current configuration of the `maven-enforcer-plugin` does not allow Java versions above 1.8.0.

``` sh
[INFO] --- maven-enforcer-plugin:1.3.1:enforce (enforce-rules) @ spring-boot-parent ---
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireJavaVersion failed with message:
Detected JDK Version: 1.8.0-5 is not in the allowed range (1.7,1.8).
[…]
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:1.3.1:enforce (enforce-rules) on project spring-boot-parent: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed. -> [Help 1]
```

Ref: http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

--- a/spring-boot-parent/pom.xml
+++ b/spring-boot-parent/pom.xml
@@ -342,7 +342,7 @@
 						<configuration>
 							<rules>
 								<requireJavaVersion>
-									<version>(1.7,1.8)</version>
+									<version>1.7</version>
 								</requireJavaVersion>
 							</rules>
 						</configuration>

